app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -1,20 +1,20 @@
 module IrsAttemptsApi
   class AttemptEvent
-    attr_accessor :jti, :iat, :event_type, :session_id, :occurred_at, :event_metadata
+    attr_accessor :jti, :iat, :event_type, :session_id, :occurred_at_ms, :event_metadata
```

Comment:
I think the attribute in Ruby is still going to be the same, `occurred_at` (a Time instance)
```suggestion
    attr_accessor :jti, :iat, :event_type, :session_id, :occurred_at, :event_metadata
```
if we did want to call it `occurred_at_ms` then I think it should be a numeric value

Reply:
Good call - thanks!